In [1]:
import numpy as np
import pandas as pd

In [2]:
data = pd.read_csv('energy.csv')

In [5]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 450 entries, 0 to 449
Data columns (total 26 columns):
 #   Column                                     Non-Null Count  Dtype  
---  ------                                     --------------  -----  
 0   Company Name                               450 non-null    object 
 1   company contact                            450 non-null    object 
 2   company email                              433 non-null    object 
 3   company phone                              450 non-null    object 
 4   Address                                    450 non-null    object 
 5   City                                       450 non-null    object 
 6   State                                      450 non-null    object 
 7   Postcode                                   450 non-null    int64  
 8   Industry                                   450 non-null    object 
 9   Industry descr                             434 non-null    object 
 10  Company Type              

450 companies are presented in the data set

In [103]:
data[data['City'] == 'Queens']['Job created'].count()

3

The total number of jobs created for businesses in Queens is 3.

In [40]:
data['company email'].nunique()

420

There are 420 different unique email domains present in the data set.


In [130]:
data['NTA'].value_counts()>5

Hunters Point-Sunnyside-West Maspeth         True
Sunset Park West                             True
Queensbridge-Ravenswood-Long Island City     True
park-cemetery-etc-Brooklyn                   True
Greenpoint                                   True
                                            ...  
Washington Heights South                    False
Forest Hills                                False
Morrisania-Melrose                          False
Ocean Hill                                  False
Co-op City                                  False
Name: NTA, Length: 79, dtype: bool